ecmascript/transforms/base/src/fixer.rs

```diff
@@ -1219,4 +1226,19 @@ var store = global[SHARED] || (global[SHARED] = {});
     identical!(issue_1781, "const n = ~~(Math.PI * 10)");
 
     identical!(issue_1789, "+(+1 / 4)");
+
+    identical!(new_member_call_1, "new (getObj()).ctor()");
+    test_fixer!(
+        new_member_call_2,
+        "new (getObj().ctor)()",
+        "new (getObj()).ctor()"
+    );
+    test_fixer!(
+        new_member_call_3,
+        "new (x.getObj().ctor)()",
+        "new (x.getObj()).ctor()"
+    );
+    identical!(new_call, "new (getCtor())");
+    test_fixer!(new_member_1, "new obj.ctor()", "new obj.ctor()");
```

Comment:
The fix doesn't change the output of the `new_member_*` tests but I figured I'd add them as negative cases. I don't exactly know why one has the trailing parens and one doesn't - it would probably be good to normalise both to the same form if we can.

Reply:
Arguments of `new` expressions are optional and the ast of swc stores if user specified parens.